beacon-chain/types/block.go

```diff
@@ -192,14 +196,27 @@ func (b *Block) isAttestationValid(attestationIndex int, aState *ActiveState, cS
 		return false
 	}
 
-	if attestation.JustifiedSlot > cState.LastJustifiedSlot() {
-		log.Debugf("attestation's last justified slot has to match crystallied state's last justified slot. Found: %d. Want: %d",
+	if !(attestation.JustifiedSlot <= cState.LastJustifiedSlot()) {
```

Comment:
Please revert this logic change.
Please prefer the positive case when writing if conditionals.

Reply:
I don't mind reverting it, but don't you think writing it this way, albeit convoluted maintains the spirit and thinking of how 2.1 was written.  It makes it easier to demonstrate that we are following the spec as written where the positive writing makes you stop and think what was intended here.

Reply:
Maybe we can change the spec 2.1 if we need parity or we can add a comment. 
We should prefer the positive case in our code. 

This code reads `if not this justified block before last justified block, then the block is not valid` which is arguably more difficult to comprehend than the original code of `if this justified slot is after the last justified slot, then the block is not valid`.

Please revert this change.

Reply:
If you'd like, we can add this comment from the spec:
```
// Verify that the justified_slot is equal to or earlier than the last_justified_slot 
// in the crystallized state.
```

Reply:
However, I still think that is redundant and doesn't provide any new information.

Reply:
+1 to Preston's feedback